### PR TITLE
Mac OSX release verification script failure fix

### PR DESCRIPTION
**Pull request by ihsandemir**
Fix to mac OSX release verification script failure (The cmake file was missing the ssl library flags).

Also a minor fix to windows release script.

### CODE REVIEW

**Comment by ihsandemir** [comment]
verify-windows

File: examples/CMakeLists.txt [hunk 1]
```diff
@@ -78,6 +78,16 @@ if (NOT("${HAZELCAST_INSTALL_DIR}x" STREQUAL "x"))
...
+    if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
+        # This is a bug in CMake that causes it to prefer the system version over
+        # the one in the specified ROOT folder. see
+        # https://stackoverflow.com/questions/58446253/xcode-11-ld-error-your-binary-is-not-an-allowed-client-of-usr-lib-libcrypto-dy
+        set(OPENSSL_ROOT_DIR ${OPENSSL_ROOT_DIR} /usr/local/Cellar/openssl@1.1/1.1.1g/)
+        set(OPENSSL_INCLUDE_DIR /usr/local/Cellar/openssl@1.1/1.1.1g/include CACHE FILEPATH "" FORCE)
+        set(OPENSSL_CRYPTO_LIBRARY ${OPENSSL_ROOT_DIR}/lib/libcrypto.dylib CACHE FILEPATH "" FORCE)
+        set(OPENSSL_SSL_LIBRARY ${OPENSSL_ROOT_DIR}/lib/libssl.dylib CACHE FILEPATH "" FORCE)
+    endif()
+
```
Comment (yemreinci) [inline examples/CMakeLists.txt:90]: We are fixing a version here. For example, the [current version](https://formulae.brew.sh/formula/openssl@1.1) of openssl in HomeBrew repositories is "1.1.1h" and we are looking at the directory "1.1.1g".

**Review by yemreinci — APPROVED** [review]

File: examples/CMakeLists.txt (hunk 1 above)
Comment (ihsandemir) [reply examples/CMakeLists.txt]: updated to latest versionç

**Review by yemreinci — APPROVED** [review]

**Comment by ihsandemir** [comment]
verify-linux

**Pull request merged**